src/utils/getArchitecture.ts

```diff
@@ -0,0 +1,19 @@
+import { Architecture } from "../types";
+import os from "os";
+
+/**
+ * Return what kind of architecture is using the machine where the build is done
+ */
+export async function getArchitecture(): Promise<Architecture> {
```

Comment:
This doesn't have to be an async function

Reply:
Fixed